apps/condo/domains/ticket/schema/TicketReportService.test.js

```diff
@@ -0,0 +1,49 @@
+const { makeLoggedInClient } = require('@condo/domains/user/utils/testSchema')
+const { createTestUser } = require('@condo/domains/user/utils/testSchema')
+const { makeLoggedInAdminClient } = require('@core/keystone/test.utils')
+const { v4: uuid } = require('uuid')
+const { createTestTicket } = require('@condo/domains/ticket/utils/testSchema')
+const { GET_TICKET_WIDGET_REPORT_DATA } = require('@condo/domains/ticket/gql')
+const { makeClient } = require('@core/keystone/test.utils')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+
+describe('TicketReportService', () => {
+    describe('User', () => {
+        it('can get ticket report with selected organization', async () => {
+            const client = await makeClientWithProperty()
+            await createTestTicket(client, client.organization, client.property)
+            const { data: { result: { data } } } = await client.mutate(GET_TICKET_WIDGET_REPORT_DATA, {
+                data: { userOrganizationId: client.organization.id, periodType: 'week' },
+            })
+            expect(data).toBeInstanceOf(Array)
+            expect(data.length).toBeGreaterThanOrEqual(1)
+            expect(data[0]).toHaveProperty('statusName')
+            expect(data[0]).toHaveProperty('currentValue')
+            expect(data[0]).toHaveProperty('growth')
```

Comment:
could you please check the whole result. (include the value of properties)

Reply:
@pahaz I can't do this becase TicketStatus name filed filled with random values